operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -46,25 +45,71 @@ const (
 // This type exists so there's no chance of pushing random strings to our progress status
 type ProgressState string
 
+const (
+	defaultConfigBuilderImage     = "datastax/cass-config-builder:1.0.0"
+	cassandra_3_11_6              = "datastax/cassandra-mgmtapi-3_11_6:v0.1.2"
+	cassandra_4_0_0               = "datastax/cassandra-mgmtapi-4_0_0:v0.1.2"
+	dse_6_8_0                     = "datastax/dse-server:6.8.0"
+	ubi_cassandra_3_11_6          = "registry.connect.redhat.com/datastax/cassandra:3.11.6"
+	ubi_cassandra_4_0_0           = "TODO"
+	ubi_dse_6_8_0                 = "registry.connect.redhat.com/datastax/dse-server:6.8.0"
+	ubi_defaultConfigBuilderImage = "registry.connect.redhat.com/datastax/cass-config-builder:1.0.0"
```

Comment:
we're gonna stick with dockerhub, no need for `registry.connect.redhat.com` because it'll be hard to test - this PR is focusing on UBI bytes, not image hosting

Reply:
Note the images on DockerHub are suffixed with `-ubi7`